test/unit/tech/html5.test.js

```diff
@@ -1,381 +1,451 @@
-var player, tech, el;
+/* eslint-env qunit */
+let player;
+let tech;
 
 import Html5 from '../../../src/js/tech/html5.js';
 import * as browser from '../../../src/js/utils/browser.js';
 import document from 'global/document';
 
-q.module('HTML5', {
-  'setup': function() {
+QUnit.module('HTML5', {
+  setup() {
 
-    el = document.createElement('div');
-    el.innerHTML = '<div />';
+    this.el = document.createElement('div');
```

Comment:
and it didn't complain about `player` and `tech`? Though, I think it breaks `player.el()` below.